### DISCUSSION TITLE
测试一下发图片

### DISCUSSION THREAD

**Discussion by zzz-Rope** [share121/inter-knot#2787] [其他]
![emotion_download_1723807643384](https://github.com/user-attachments/assets/87919451-61d6-411f-a1cc-19488cddf17d)
![Screenshot_20240814_173403](https://github.com/user-attachments/assets/a94a98b4-acdb-438e-9703-ba1ed68e3fca)

**Comment by demolute** [comment]
请问是枫丹的芙宁娜女士吗？

**Comment by zzz-Rope** [comment]
我不是已经删了吗<img src="https://img-static.mihoyo.com/communityweb/upload/289c384786292056f6ff640ed67b44e7.png" alt="唔" width="75" >

**Comment by zzz-Rope** [comment]
为什么我删了还能刷新出来？怎么删啊？
